tests/StorageApiTestCase.php

```diff
@@ -391,4 +392,32 @@ protected function waitForFile($fileId, $sapiClient = null)
             sleep(pow(2, $tries));
         }
     }
+
+    protected function createTableWithRandomData(string $tableName, int $rows = 5, int $columns = 10, $charsInCell = 20): string
```

Comment:
Tuhle metodu když tak můžu vyhodit samostatného PR. Každopádně mi příjde hodně "handy" :)

Reply:
Aby byla handy, tak je IMHO potřeba, aby byla rozdělená na dvě části - vygenerovat data a pak vygenerovat tabulku s těma datama. Jinak se ti bude blbě dělat assert, když je to pokaždé jiné.

Reply:
Kdo bude potřebovat, tak si udělá další metodu, která bude přebírat konkrétní data. Já často potřebuju fakt jenom náhodný data.